content/2.0.html.haml

```diff
@@ -34,12 +34,18 @@ title: "Jenkins 2.0"
 
 
 %h1
-  Jenkins 2.0 preview
+  Jenkins 2.0 Overview
 
 %p
-  %strong
-    Jenkins 2.0
-  is an upcoming major version of Jenkins with a number of usability enhancements and new features.
+  - # Putting this section into markdown to make it easier to mark up
+  :markdown
+    Jenkins is an open source automation server with an unparalleled plugin
```

Comment:
This is still stupid. One would now think this page is generically about Jenkins, rather than what's new in 2.0. And for that, it's missing a lot of highlights.